vulcan/docker-compose.yml

```diff
@@ -10,12 +10,13 @@ x-base-etna-service:
     - vulcan-log-dir:/app/log:rw
     - vulcan-tmp-dir:/app/tmp:rw
     - vulcan-public-dir:/app/public:rw
-    - vulcan-data-dir:/app/data:rw
     - vulcan-opt-dir:/usr/opt:rw
     - ../etna:/etna:delegated
+    - /var/run/docker.sock:/var/run/docker.sock:ro
   environment: &base-etna-env
     'VULCAN_ENV': 'development'
     APP_NAME: 'vulcan'
+    HOST_DIR_MAP: "/app=${PWD}/../vulcan"
```

Comment:
Data directories are mounted from the host, so we need the mapping from the host directory to the container directory to enable sending the correct docker commands.